README.md

```diff
@@ -1,18 +1,24 @@
-# UIKit-cross-platform
-
 [![CircleCI](https://circleci.com/gh/flowkey/UIKit-cross-platform.svg?style=shield&circle-token=2bc60653f4bb369754b04d97d324d9ba2bee6c6b)](https://circleci.com/gh/flowkey/UIKit-cross-platform)
 [![Swift4.1](https://img.shields.io/badge/swift-4.1-orange.svg?style=flat)](https://swift.org/)
 [![Platforms](https://img.shields.io/badge/platform-Android%20%7C%20macOS-lightgrey.svg)](https://swift.org/)
 [![License](https://img.shields.io/badge/license-MIT-71787A.svg)](https://tldrlegal.com/license/mit-license)
+# UIKit-cross-platform
+
+### The missing part to make your Swift UI Code work on Android
```

Comment:
Let’s keep the heading levels consistent (by that I mean a first-level heading should contain second level headings etc - markdown should still be semantic)

Also, I think this sentence is a bit long. What about:

“## Your Swift UI Code on Android”

To be honest though I don’t think we need this subheading. Personally I would consider removing it but I’m sure you had a good reason to put it in (I guess for search exposure)